lib/confluence/aggressive_removal_data.es6.js

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.CLASS({
```

Comment:
I think this contains some duplicate information about the related browsers, but it's not urgent that we rework that right now.